### ISSUE TITLE
Add fee to workshop costs in FAQ

### ISSUE THREAD

**Issue by dlebauer** [swcarpentry/site#892]
The FAQ has a workshop costs section:

```html
  <dt id="workshop-costs">Are workshops free?</dt>
  <dd>
    <p>
      Some workshops charge $20-$40 for registration,
      primarily because it reduces the number of people who register but not attend
      from 20-25% to roughly 5%.
    </p>
  </dd>
```

But there is no mention of the suggested donation. I'd suggest adding something here.

**Comment by wking** [comment]
On Tue, Mar 17, 2015 at 02:34:55PM -0700, David LeBauer wrote:
> But there is no mention of the suggested donation.

In fact, we have a required (but occasionally waived) fee for
workshops organized by SWC admins (point 3 in [1]).

[1]: https://github.com/swcarpentry/board/blob/6ada614feae7a7874765b5a313a924a9891583d8/workshops.md#operations

**Comment by gvwilson** [comment]
Closed by 3a3b2d984afa9429253bfa1ea0a5bbc3e654bf3c
